### PR TITLE
fix

### PR DESCRIPTION
**Pull request by alibazregar**

### CODE REVIEW

File: src/components/modules/PartBaseModule.ts [hunk 1]
```diff
@@ -36,15 +36,21 @@ export default abstract class PartBaseModule<
           ? questionPart.caption
           : //@ts-ignore
```
Comment (Qamsari) [inline src/components/modules/PartBaseModule.ts:37]: Why we must use `//@ts-ignore`?
It is not good manners to handle exceptions. I think

File: src/components/workspace/assets/style.css [hunk 2]
```diff
@@ -213,27 +213,28 @@
 [data-get-json-for-download] {
   display: none;
 }
-[data-details]
+#form-details-schema
```
Comment (Qamsari) [inline src/components/workspace/assets/style.css:216]: Please use `data-` for select business base element for apply styles

**Review by Qamsari — COMMENTED** [review]
Please see mu issues.

File: src/components/modules/PartBaseModule.ts (hunk 1 above)
Comment (alibazregar) [reply src/components/modules/PartBaseModule.ts:37]: how can we handle the parts that are not in basiscore models?

**Pull request merged**